packages/edit-post/src/components/layout/offline-status.native.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { Text } from 'react-native';
+import Animated, {
+	useSharedValue,
+	useAnimatedStyle,
+	withTiming,
+} from 'react-native-reanimated';
+
+/**
+ * WordPress dependencies
+ */
+import { Icon } from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+import offlineIcon from './offline-icon';
+
+const OfflineStatus = () => {
+	const translateY = useSharedValue( -10 );
+	const slideIn = () => {
+		translateY.value = withTiming( 0, { duration: 100 } );
+	};
+
+	useEffect( () => {
+		slideIn();
+	}, [] );
```

Comment:
Do we need to return a clean up function or it safe for this animation to be invoked if the component is un-mounted?

Reply:
In testing, I didn't feel this animation actually added much value. I've removed it for now -- we could test re-introducing it when this component is wired up to the connection subscription hook, or seek design feedback.